roles/confluent.kafka/templates/server_ssl.properties.j2

```diff
@@ -1,5 +1,5 @@
 # Maintained by Ansible
-listeners=SSL://:{{broker.config.port}}
+listeners=SSL://:{{kafka_port}}
```

Comment:
Recommend leaving in 'broker' as kafka is not a component but the platform name.